components/Modules/module-detail-page.tsx

```
@@ -726,15 +726,21 @@ export default function ModuleDetailWrapper({
               </div>
             ) : (
               <h1
-                className="text-xl font-bold cursor-pointer hover:bg-muted/50 px-2 py-1.5 rounded"
+                className="text-xl cursor-pointer hover:bg-muted/50 px-2 py-1 rounded"
                 ref={titleEditRef}
                 onClick={() => setIsEditingTitle(true)}
+                tabIndex={0}
+                onKeyDown={(e) => {
```

Biome (1.9.4)

[error] The HTML element h1 is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)